README.md

```diff
@@ -1 +1,56 @@
-# Getting started with Cisco IOS-XE Smart licensing APIs
\ No newline at end of file
+# Getting Started with Cisco IOS-XE Smart licensing APIs
+
+Sample code in Python that shows how to call IOS-XE SLP REST APIs. You can automate the usage reporting workflows using these APIs.
+
+For information about Smart Licensing Using Policy (SLP), see [About Smart Licensing Using Policy](ios-xe-smart-licensing-api/intro.md#about-smart-licensing-using-policy).
+
+## Use Case Description
+
+For information about the purpose of each currently available API, see [About IOS-XE SLP REST APIs](ios-xe-smart-licensing-api/intro.md#about-ios-xe-slp-rest-apis).
+
+Future enhancements include an API to return a Smart Licensing Authorization Code (SLAC).
+
+## Installation
+
+Python must be installed. Further the following Python packages are required:
+* os
+* shutil
+* json
+* xmltodict
+* requests
+* yaml
+* tarfile
+
+## Configuration
+Configure the following where? we have to check CSSM API docs to know this and then add. 
+
+client_id: 55f65e59-df52-4073-bb12-4268d5167019 (the user has to enter their own;  will enquire how this should be done)
+client_secret: 78962244-9f2f-4e50-b233-f92adedd1e33 (the user has to enter their own; will enquire how this should be done)
+username: CSSM log in username (update config.yaml with dummy username)
+password: CSSM log in password (update config.yaml with dummy password)
+
+
+## Usage
```

Comment:
Run the script by doing "python sl-api-demo.py"
This will give a choice of four operations.
1: Get the list of SA/VA associated with the CSSM username.
2: Collect usage (RUM) reports from the devices mentioned in the config.yaml file
-- Now these reports have to be manually uploaded into CSSM, and ACK for them has to be downloaded.
-- Script support for that can be included in a future revision of the script.
3: Send the ACK file to a device from the config.yaml file
4: Send a policy file to a device from the config.yaml file

Reply:
This comment needs further clarification Sujay. I will send you an email about this.